src/main/java/org/springframework/cloud/dataflow/module/deployer/cloudfoundry/ApplicationModuleDeployer.java

```diff
@@ -163,6 +164,23 @@ public void run() {
 		return moduleDeploymentId;
 	}
 
+	private int deduceMemory(Map<String, String> deploymentProperties) {
+		String override = deploymentProperties.get("cloudfoundry.memory");
```

Comment:
OK, cool. Yes, units would be a good thing. Maybe we can just piggyback on the already supported format in the manifest.  https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#disk-quota